addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/launch-product/launch-product.js

```diff
@@ -356,6 +363,35 @@ class LaunchProduct extends StepBase {
     }
     return result.LaunchPathSummaries[0];
   }
+
+  /**
+   * Check if the resolved input parameter contains a static namespace param. If so, augments the namespace to begin with 
+   * 'analysis-' for permissions purposes (if it does not already start with that) and to end with a unique datetime string 
+   * so Cloudformation doesn't create duplicate stacks for separate workspaces.
+   * 
+   * @param resolvedInputParams 
+   * @param datetime 
+   * @returns {Promise<{namespaceParam:string, namespaceIndex:string}[]>}
+   */
+  async getNamespaceAndIndexIfNecessary(resolvedInputParams, datetime){
+    const namespaceIndex = resolvedInputParams.findIndex(element => element.Key === 'Namespace');
+    let namespaceParam = resolvedInputParams[namespaceIndex].Value;
+    console.log(namespaceParam);
```

Comment:
If you are logging something try to add more description. Something along the lines of `Current namespace in resolvedInputParams is ${namespaceParam}`

Reply:
I actually meant to take this out--thanks for the advice and for flagging it for me!